msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -186,13 +203,14 @@ internal TokenCache UserTokenCache
         /// (a list of known B2C authorities cannot be maintained by MSAL.NET). This property can be set just after the construction of the application
         /// and before an operation acquiring a token or interacting with the STS.
         /// </summary>
+        // TODO:  [Obsolete("ValidateAuthority is no longer used.  You should provide authority information via the configuration builder.")]
```

Comment:
>Obsolete [](start = 19, length = 8)

removing these if public is a breaking change. ValidateAuthority is used in .net, we are calling the open id metadata end-point which afaik Android doesn't do as they require config of these parameters. Not sure that is the way we should go though.